packages/types/src/config.ts

```diff
@@ -129,10 +129,15 @@ export interface SlidevConfig {
    * @default 'https://cdn.jsdelivr.net/gh/slidevjs/slidev/assets/favicon.png'
    */
   favicon: string
+  /**
+   * Options for shortcuts
+   * @default {}
+   */
+  shortcuts: ResolvedShortcutOptions
```

Comment:
While I do agree to make it an option would help on future extensive, this option might mislead users that you may config shortcuts in the frontmatter.

I guess maybe, instead, we could pass the default shortcut array as an argument to `setupShortcuts` so the user could mutate it or disable completely setting `arr.length = 0`. Wdyt?

Reply:
Uhh, that's even better! Great idea 👍 
Will refactor ☺️